### ISSUE TITLE
can't access property "x", max2 is undefined error

### ISSUE THREAD

**Issue by dilawar** [vUdav/vue2-leaflet-locatecontrol#9]
My issue is very similar to this question on SO

https://stackoverflow.com/q/61923621/1805129

**Comment by Soug-crypto** [comment]
Do you get the error when you clone and npm run example? I didn't.

**Comment by erfanSW** [comment]
This is solution :
import Vue2LeafletLocatecontrol from "vue2-leaflet-locatecontrol/Vue2LeafletLocatecontrol";

**Comment by Triloworld** [comment]
Nuxt testing show same error. Next solution @vUdav will create another one in my enviroment as css isn't properly imported
- "_Error: Can't resolve '~leaflet.locatecontrol/dist/L.Control.Locate.css'_"

However, It will work after copying to plugin vue file and commented css line.

Example Nuxt config:
```js
 css: [
  'leaflet.locatecontrol/dist/L.Control.Locate.css'
 ],
 plugins: [
  { src: "~/plugins/vue2-leaflet-locatecontrol.js", mode: 'client' },
 ]
```
vue2-leaflet-locatecontrol.js:
```
import Vue from 'vue'
import Vue2LeafletLocatecontrol from "./Vue2LeafletLocatecontrol.vue";

Vue.component('v-locatecontrol', Vue2LeafletLocatecontrol);
```
File copy from ```vue2-leaflet-locatecontrol/Vue2LeafletLocatecontrol``` to ```./plugins/Vue2LeafletLocatecontrol.vue```
**Commented line with css**

**Comment by dilawar** [comment]
> Do you get the error when you clone and npm run example? I didn't.

@Soug-crypto No longer using this library in my project. So really can't say.

**Comment by ThunderBirdsX3** [comment]
https://github.com/vUdav/vue2-leaflet-locatecontrol/issues/3#issuecomment-522104281

**Comment by vUdav** [comment]
I fix this problem in v1.0.2
